CHANGELOG.md

```diff
@@ -29,7 +29,6 @@
 - Do not set MeterProvider to global otel (#5146)
 - Make `InstrumentationLibrary<signal>ToScope` helper functions unexported (#5164)
 - Remove Log's "ShortName" from logging exporter output (#5172)
-- `exporter/otlp`: Retry RESOURCE_EXHAUSTED only if the server returns RetryInfo (#5147)
```

Comment:
Looks like this PR includes some unrelated changes. Can you please remove them? Also please add a changelog entry

Reply:
sure, caused by wrong revert click after rebase